PrioritiesSprint1.md

```diff
@@ -0,0 +1,81 @@
+
+# Develop Pathfinder Query
+## Pre-development work:
+- Determine 2-3 specific use cases for a Pathfinder query that provide value to a user AND have known inputs/expected outputs. 
+- Use those specific use cases as the basis for discussion. Brainstorm to identify knowns and unknowns (eg. we know TRAPI and UI changes will be needed - what else needs to be considered?)
+- Identify next steps and map out what can be completed in the upcoming development cycles
+- Discuss TRAPI changes and either outline a plan for those changes AND/OR set planning goals for TRAPI for Octopus Backend Development cycle
+- Discuss UI changes and either outline a plan for those changes AND/OR set planning goals for upcoming development cycles
+
+# Add Phenotypes (HPO terms) to MVP1
+## Work in CI (Lobster)
+- Push to CI
+## Work in Test (Pre-Lobster)
+- Push to Test
+- Test for controversial items
+- Test for child terms
+
+# Scalability
+## Pre-Development
+- TRAPI defines syntax for bypassing caches in tools
+## Work in CI (Lobster)
+- Send concurrent queries (MVP1 & 2) in batches of 5, 10, 15, 20, 25, 30
+## Work in Test (Pre-Lobster)
+- Send concurrent queries (MVP1 & 2) in batches of 5, 10, 15, 20, 25, 30
+
+# Treats Refactor
+## Pre-Development
+- ARAs review the changes needed to continue use of treats
+- Document usage of "Drug" vs "Procedure" vs "Treatment" categories
+- Provide edge examples in Biolink documentation for "treats" predications
+- Complete the "treats" refactor source transformation spreadsheet
+## Work in CI (Lobster)
+- KPs implement Biolink 4.x including the refactoring of treats edges
+
+# CQS
+## Work in CI (Lobster)
+- Push initial workflows (Paths A, B, E) to TEST (end of cycle) contingent on CI tests
+- Hook up CI instance of CQS to ARS in CI
+- Direct CQS CI tests using test assets with new Q-A pairs
+- ARS CI tests using Information Radiator
+- Extend CQS to support new MVP workflows
+- Develop valid TRAPI queries and deposit to CQS repo
+- Identify relevant tests assets and deposit to Testing Team's G-sheet/repo
+- Direct CQS tests in DEV using test assets
+
+# MultiCURIE Queries
+## Pre-Development
+- Determine 2-3 specific use cases for a MultiCURIE query that provide value to a user AND have known inputs/expected outputs.
+- Use those specific use cases as the basis for discussion. Brainstorm to identify knowns and unknowns (eg. we know TRAPI and UI changes will be needed - what else needs to be considered?)
+- Identify next steps and map out what can be completed in the upcoming development cycles
+- Discuss TRAPI changes and either outline a plan for those changes AND/OR set planning goals for TRAPI for Octopus Backend Development cycle
+- Discuss UI changes and either outline a plan for those changes AND/OR set planning goals for upcoming development cycles
+
+# New Merged Scores
+## Pre-Development
+- Address g() and f() for ordering / confidence score ties for MVP1 and MVP2
```

Comment:
I strongly suggest that you clarify the terminology here and refer to "Confidence Score" and "Ordering Score" for consistency and to avoid continued confusion. Alternatively, you can refer to "Confidence Score (aka, g()-score)" and "Ordering Score" (aka, f()-score, Sugeno, Sugeno Integral).